portability-api-launcher/src/main/java/org/datatransferproject/launcher/monitor/events/EventCode.java

```diff
@@ -15,6 +15,9 @@
   public static final EventCode WORKER_JOB_FINISHED = new EventCode("WORKER_JOB_FINISHED");
   public static final EventCode WORKER_JOB_STARTED = new EventCode("WORKER_JOB_STARTED");
 
+  public static final EventCode WATCHING_SERVICE_JOB_ERRORED = new EventCode("WATCHING_SERVICE_JOB_ERRORED");
```

Comment:
Why do we want to specify the WATCHING_SERVICE here? Instead of using WORKER_JOB_PREEMPTED or WORKER_JOB_ERRORED?

Reply:
these are added to distinguish events being logged by WORKER_JOB threads and WATCHING_SERVICE threads. 
Existing event, WORKER_JOB_CANCELLED is left untouched so that existing metrics are left unmatched.